extensions/blocks/podcast-player/style.scss

```diff
@@ -119,9 +119,20 @@ $player-float-background: $light-gray-200;
 	}
 
 	.jetpack-podcast-player__audio-player {
+		height: 40px; /* the same as .mejs-container */
 		margin-bottom: $player-grid-spacing;
 	}
 
+	&.is-default .jetpack-podcast-player__audio-player {
+		display: none;
+	}
+
+	.jetpack-podcast-player--audio-player-loading {
+		height: 10px;
```

Comment:
Should we use a variable for this `10px` value to document it at the top of the file?